addons/turbowarp-player/addon.json

```diff
@@ -48,6 +49,12 @@
       ],
       "default": "link"
     },
+    {
+      "name": "Automatically open/replace",
+      "id": "auto",
+      "type": "boolean",
+      "default": false
+    },
```

Comment:
As Samq64 said, I think we may need to consider that people may have set the addon to open a new tab. How would that work out? It's intrusive to always open a new tab whenever you visit a project, especially if you just want to edit it or see a comment someone left.

Reply:
I would say just don't show the toggle when the new tab mode is selected.

Reply:
> I would say just don't show the toggle when the new tab mode is selected.

Good idea. We could also display an icon or label-button on the stage to play the project on TurboWarp, though, that kind of defeats the purpose of it being automatic.